cmd/fetch_ca_cert/fetch_ca_cert.go

```diff
@@ -188,19 +188,24 @@ func main() {
 
 	client, err := privateca.NewCertificateAuthorityClient(context.Background())
 	if err != nil {
+		client.Close()
 		log.Fatal(err)
 	}
 	parsedCerts, err := fetchCACertificate(context.Background(), *gcpCaParent, *kmsKey, *tinkKeysetPath, *tinkKmsKey, client)
```

Comment:
I think we can probably just put the `defer client.Close()` call here after the error check once and then remove all the other calls to `client.Close()`

Reply:
IIRC the problem is that log.Fatal ends execution immediately so the deferral doesn't happen. I think the linter complains about the lack of closure in that case. I ran into this with closing files in other tests

Reply:
No, this is due to the nature of log.Fatal. It won't run the defer.